helm/superset/templates/_helpers.tpl

```diff
@@ -73,10 +104,7 @@ CACHE_CONFIG = {
       'CACHE_TYPE': 'RedisCache',
       'CACHE_DEFAULT_TIMEOUT': 300,
       'CACHE_KEY_PREFIX': 'superset_',
-      'CACHE_REDIS_HOST': env('REDIS_HOST'),
-      'CACHE_REDIS_PORT': env('REDIS_PORT'),
-      'CACHE_REDIS_PASSWORD': env('REDIS_PASSWORD'),
-      'CACHE_REDIS_DB': env('REDIS_DB', 1),
+      'CACHE_REDIS_HOST': redis_cache_config,
```

Comment:
how about using `CACHE_REDIS_URL` here instead?

Reply:
That is a good suggestion, why didn't I think about it. 🤦🏻‍♂️  
Redis cache URL is already generated by template based on values.  
we can use it instead of creating redis.Redis object

Reply:
you need to bump the chart version also

Reply:
User `CACHE_REDIS_URL` and bumped up Chart Version..